build.gradle

```
@@ -32,24 +31,29 @@ dependencies {
     implementation(group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1')
     implementation(group: 'javax.el', name:'javax.el-api', version:'3.0.0')
     // Groovy
-    implementation(group:'org.codehaus.groovy', name:'groovy-all', version:'3.0.8')
+    implementation (group: 'org.apache.groovy', name: 'groovy-all', version: '4.0.15', ext: 'pom')
```

It looks like Groovy is only used on tests. This could be changed to `testImplementation` scope:
```suggestion
    testImplementation (group: 'org.apache.groovy', name: 'groovy-all', version: '4.0.15', ext: 'pom')
```